azure-pipelines.yml

```diff
@@ -247,6 +228,30 @@ jobs:
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check Licence Headers
 
+- job: slow_lints
+  displayName: Perform Slower Lints
+  dependsOn: lint
+  pool:
+    vmImage: ubuntu-18.04
```

Comment:
We probably should keep it in 16.04 and then switch over in one go in #3709

Reply:
Duuur. that was my intention